### DISCUSSION TITLE
When is openbao release happening?

### DISCUSSION THREAD

**Discussion by slice-nishantk** [openbao/openbao#147] [General]
We're developing a solution that requires Vault's transit secrets engine. We came across openbao as an open source alternative to hashicorp's vault and are really keen to use openbao as our core vault. When is the first openbao release going to happen?

**Comment by joewxboy** [comment]
@slice-nishantk That's a great question.  it would be nice to have a page on the wiki with a provisional set of milestone targets that we're aiming for.  The Open Horizon project is an OpenBao adopter, and we plan to begin integrating OpenBao's Alpha release when that is ready.  @naphelps do we have the rough timing determined for that?

@slice-nishantk If you or some co-workers could begin participating on our weekly community calls, that may give you a forum to ask questions when the whole team is present.  And to ensure that OpenBao meets your needs, it would be great to get you connected to the volunteers that are working on that feature, to collect any requirements that you may have, and to see if your team can help out in any way.  We're not a large project with paid staff working on a commercial product, so our efforts and timelines are affected by who is able to lend a hand.  The more hands involved, the quicker we can work on issues. :)

**Comment by naphelps** [comment]
Hi @slice-nishantk. It is too early at this time to make any commitments to when OpenBao will have its first GA release. Currently, we are working towards an Alpha release by end of March (tentative).

**Reply by naphelps** [reply]
As Joe mentioned the best place to follow information about timelines and releases is our weekly community calls and our chat channels.

**Reply by juliadin** [reply]
As a member of the IT security team at another company evaluating adoption of openbao, would it be ok to just try to join the weekly calls or are those restricted in any way? I'd hate to barge in (would stay quiet though, I am currently trying to stay up to date with what you are up to with some excitement :)

**Reply by joewxboy** [reply]
@juliadin Thanks for asking.  These are public meetings, and your company is a potential adopter, so the calls are the perfect place to interact with the whole community at once (DISCLAIMER: but keep in mind that they are recorded, and we need to follow anti-trust policies, so no discussions about confidential commercial product timelines, pricing, collusion, or anything else prohibited by law).  

Additionally, feel free to ask questions on our chat rooms at https://chat.lfx.linuxfoundation.org/ in the rooms beginning with "openbao" in the title.

**Comment by naphelps** [comment]
OpenBao has released its first Alpha 2024-04-01.

https://github.com/openbao/openbao/releases/tag/v2.0.0-alpha20240329

**Comment by cipherboy** [comment]
I'm happy to announce OpenBao's beta release was initiated yesterday: https://github.com/openbao/openbao/releases/tag/v2.0.0-beta20240618 

Per last week's community call, the GA is tentatively scheduled for four weeks out, on July 16th.

**Comment by cipherboy** [comment]
Just to wrap this one up, the GA release was [announced on July 17th](https://github.com/orgs/openbao/discussions/415)
